src/pyper/env.py

```diff
@@ -0,0 +1,78 @@
+"""Load environment variables from a `.env` file."""
+import os
+from pathlib import Path
+
+_EQUALS = "="
+_NEW_LINE = "\n"
```

Comment:
What about Windows line endings!? \r\n

Reply:
Thanks for noticing. I wasn't aware of that when I implemented this functionality. However, the built-in `open` function saved my a*: [Python Built-in Functions](https://docs.python.org/3/library/functions.html#open:~:text=newline%20determines%20how,the%20given%20string).

TL;DR: Windows' new lines `\r\n` are by default replaced with `\n`.

Reply:
Maybe I knew this at some point, but it wasn't in my working set. Thanks!